src/include/duckdb.h

```diff
@@ -155,6 +159,16 @@ typedef struct {
 	int32_t micros;
 } duckdb_time_struct;
 
+//! TIME_TZ is stored as 40 bits for int64_t micros, and 24 bits for int32_t offset
+typedef struct {
+	uint64_t micros;
```

Comment:
Can we call this "bits" to match the internal type - micros is actually not correct.